source/support/topics/paas/how-to-deploy-from-private-repository.md

```diff
@@ -0,0 +1,35 @@
+First you'll need to have created an app on Aptible. You will still need to use git, since we need to have a Procfile, used to define the app's services and their commands. If your entire app is prebuilt in the image, you do not need a Dockerfile.
+
+Next, use the Aptible CLI's `aptible config:set` command to set a few environment variables:
+
+  * `APTIBLE_DOCKER_IMAGE` - The name of the image to pull, in the format `registry_host/repo/image_name:tag`. Repo and image name are required. Tag is optional and will default to `latest`. 
+  If you are not using Docker Hub (if for example you are using quay.io) then the registry host name is also required.  
+  * `APTIBLE_PRIVATE_REGISTRY_HOST` - The private registry host to pull from. Required when pulling from a private registry. Defaults to `docker.io`
```

Comment:
That's definitely a worthy topic for discussion! Maybe we can schedule a quick session later today or tomorrow?